schemas/oic.sec.roletype.json

```diff
@@ -9,10 +9,12 @@
       "properties": {
         "authority": {
           "type": "string",
+          "readOnly": true,
```

Comment:
Again I'm not sure it makes sense to make a data-type element read-only... access mode should depend on the Property of the Resource in question, so the /roles Resource could have a read-only "roles" Property, for example, but this is like saying "int" is read-only just because the Resource that uses it makes it read-only, I think?

Reply:
i am not sure if we can put readonly at the roles Resource level. This will imply it will only support GET. I will try to discuss with Data Model gurus and update you.